shared/js/background/classes/custom-rules-manager.js

```diff
@@ -0,0 +1,40 @@
+import settings from '../settings.es6'
+const tabManager = require('../tab-manager.es6')
+
+import {
+    generateDNRRule
+} from '@duckduckgo/ddg2dnr/lib/utils'
+const { getNextSessionRuleId } = require('../dnr-session-rule-id')
+
+function getInverseRules (action) {
+    const inverseCustomRules = JSON.parse(JSON.stringify(settings.getSetting('inverseCustomRules')))
+    return inverseCustomRules?.filter(rule => rule.customAction === action)
+}
+
+export function enableInverseRules (customAction, tabId) {
```

Comment:
Please can you add a JSDoc comment for this? Then as a bonus tslint will check the input types are correct.